dev/devicelab/lib/command/test.dart

```diff
@@ -24,6 +24,11 @@ class TestCommand extends Command<void> {
           'settings in the test case, and will results in error if no device\n'
           'with given ID/ID prefix is found.',
     );
+    argParser.addFlag(
+      'exit',
+      help: 'Exit on the first test failure. Currently flakes are intentionally (though '
+            'incorrectly) not considered to be failures.',
```

Comment:
why incorrectly?

Reply:
This is copy-pasted from the `--exit` flag in `run.dart`. This verbiage was introduced in a [commit from @Hixie](https://github.com/andrewkolos/flutter/commit/57f097b532e6afeb08e3256a79ff3009a1210281#diff-cfb3295ba5fb046b811746f29aa08bce2188594d44744c2720e39da5016d5292R72). I interpret this as an opinion that tasks should be considered failing if they fail on their first run, regardless of whether or not they pass after retrying.

Reply:
flakes are by definition failing tests (that sometimes don't report their failure).

Reply:
ahh, i see